python/paddle/fluid/tests/unittests/test_imperative_layer_children.py

```diff
@@ -43,7 +44,7 @@ def forward(self, inputs):
 
 
 class TestLayerChildren(unittest.TestCase):
-    def test_apply_init_weight(self):
```

Comment:
Make sure all these tests run in eager mode with correct res

Reply:
Done.